src/App.tsx

```diff
@@ -1,173 +1,43 @@
-import { Loader } from './components/Loader';
-
+import { NavLink, Outlet } from 'react-router-dom';
 import './App.scss';
-
-export const App = () => (
-  <div data-cy="app">
-    <nav
-      data-cy="nav"
-      className="navbar is-fixed-top has-shadow"
-      role="navigation"
-      aria-label="main navigation"
-    >
-      <div className="container">
-        <div className="navbar-brand">
-          <a className="navbar-item" href="#/">
-            Home
-          </a>
-
-          <a
-            className="navbar-item has-background-grey-lighter"
-            href="#/people"
-          >
-            People
-          </a>
-        </div>
-      </div>
-    </nav>
-
-    <main className="section">
-      <div className="container">
-        <h1 className="title">Home Page</h1>
-        <h1 className="title">People Page</h1>
-        <h1 className="title">Page not found</h1>
-
-        <div className="block">
-          <div className="box table-container">
-            <Loader />
-
-            <p data-cy="peopleLoadingError" className="has-text-danger">
-              Something went wrong
-            </p>
-
-            <p data-cy="noPeopleMessage">
-              There are no people on the server
-            </p>
-
-            <table
-              data-cy="peopleTable"
-              className="table is-striped is-hoverable is-narrow is-fullwidth"
+import classNames from 'classnames';
+
+export const App = () => {
+  const isActiveLink = ({ isActive }: { isActive: boolean }) => (
```

Comment:
You are returning a class name for styling, not a boolean value, so you should name the variable differently.
```suggestion
  const activeLinkClass = ({ isActive }: { isActive: boolean }) => (
```
Also, this function should be moved outside the component.